### ISSUE TITLE
getSerializedATN() const' marked 'override', but does not override

### ISSUE THREAD

**Issue by riccardodebenedictis** [DanMcLaughlin/antlr4#14]
Dear all,

I have a minor issue with the generated code:

the header of the generated lexer (in my case ORLexer.h) contains the following line

`virtual const std::vector<uint16_t> getSerializedATN() const override;`

however, the compiler complains saying that

```
In file included from parser/ORLexer.cpp:5:0:
parser/ORLexer.h:37:39: error: 'virtual const std::vector<short unsigned int> ORLexer::getSerializedATN() const' marked 'override', but does not override
   virtual const std::vector<uint16_t> getSerializedATN() const override;
                                       ^
```

it was sufficinet, for me, to remove the `override` from the generated code..

hope that helps!

again, many thanks for your precious work!

**Comment by Nic30** [comment]
There  is just missing "const"

**Comment by mike-lischke** [comment]
This issues has already been fixed in revision [2352ff0].
